hack/validation/test_release.py

```diff
@@ -0,0 +1,181 @@
+import csv
+import pytest
+
+CSLB_PATH = 'tsv_output/cslb.tsv'
+HLES_DOG1_PATH = 'tsv_output/hles_dog_1.tsv'
+HLES_DOG2_PATH = 'tsv_output/hles_dog_2.tsv'
+
+
+@pytest.fixture(scope='session')
+def cslb_data():
+    return parse_file(CSLB_PATH)
+
+
+@pytest.fixture(scope='session')
+def hles_dog1_data():
+    return parse_file(HLES_DOG1_PATH)
+
+
+@pytest.fixture(scope='session')
+def hles_dog2_data():
+    return parse_file(HLES_DOG2_PATH)
+
+
+def parse_file(path):
+    rows = []
+    with open(path) as f:
+        reader = csv.DictReader(f, delimiter='\t')
+        for row in reader:
+            rows.append(row)
+
+    return rows
+
+
+def test_cslb_date(cslb_data):
+    for row in cslb_data:
+        assert '2021' not in row['cslb_date']
```

Comment:
not sure how much effort we want to put into this validation, but there's two things we can do to make failures easier to debug:

1. use unittest as our testing library (we can still run via pytest) and wrap these tests in a class that extends `unittest.TestCase` so we can use unittest's assertion methods, e.g. `self.assertNotIn('2021', row['cslb_date'])`, which produce failure messages with more context, rather than `assert`, which just raises an `AssertionError` saying that the assertion was false.
2. add some info about the current row to the failure message so we get a more actionable error message than `'2021' found in '2021-02-02'`

Reply:
I'll shift to unittest and add the descriptive msg, good call.